### DISCUSSION TITLE
Yet another Web Frontend: ChatSydney, no Node.js required.

### DISCUSSION THREAD

**Discussion by ghost** [acheong08/EdgeGPT#393] [Show and tell]
https://github.com/InterestingDarkness/ChatSydney/tree/master
Written in ~pure/vanilla JavaScript~, no Node.js required!
Now uses React to ease state management https://github.com/InterestingDarkness/ChatSydney , still no Node.js required.

**Comment by Nu11Const** [comment]
Are you deploy yourself?

**Reply by ghost** [reply]
`s/deploy/develop/`
GPT-4 helped me
Some chat history (in Chinese):
[前端处理连续返回消息 (Front-end handles continuous response).pdf](https://github.com/acheong08/EdgeGPT/files/11440143/Front-end.handles.continuous.response.pdf)

**Comment by Nu11Const** [comment]
I think this GUI are not beautful.Can you add some css?

**Reply by ghost** [reply]
completed https://github.com/InterestingDarkness/ChatSydney/commit/f2799899f8f736e50279511c54634757ab2bda30
